modules/machineset-vsphere-data-disks.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-vsphere.adoc
+// * machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-vsphere.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machineset-vsphere-data-disks_{context}"]
+= Configuring data disks by using machine sets
+
+{product-title} clusters on {vmw-first} support adding up to 29 disks to the virtual machine (VM) controller.
+
+:FeatureName: Configuring {vmw-short} data disks
+include::snippets/technology-preview.adoc[]
+
+By configuring data disks, you can attach disks to VMs and use them to store data for etcd, container images, and other uses.
+Separating data can help avoid filling the primary disk so that important activities such as upgrades have the resources that they require.
+
+[NOTE]
+====
+Adding data disks attaches them to the VM and mounts them to the location that {op-system} designates.
+To mount the data disks to a specific location, you must configure each machine to use the data disks according to your needs.
```

Comment:
This needs more actionable detail. I'm not really familiar with machine config stuff, but I am guessing this might involve some existing or new procedure in [this area of docs](https://93585--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_configuration/machine-configs-configure)?

Reply:
I was trying to leave this vague since we do not do anything else with it at this time.  If it reduces confusion, we can remove the last sentence.  I am working on examples of what MachineConfig will look like for taking the attached disk and putting it somewhere in the OS for special needs, but this would be not necessarily need for this portion of the feature.